### ISSUE TITLE
[minillm] does qwen2 support model parallelism?

### ISSUE THREAD

**Issue by 01yingxuan** [microsoft/LMOps#318]
When performing SFT on the Qwen2.5 32B model, I encountered an OOM (Out of Memory) issue. I would like to ask if the Qwen series of models supports model parallelism during SFT?

**Comment by t1101675** [comment]
Yes. You can use Model Parallelism just like other models by setting 
```bash
OPTS+=" --model-parallel"
OPTS+=" --model-parallel-size ${MP_SIZE}"
```

**Comment by 01yingxuan** [comment]
> Yes. You can use Model Parallelism just like other models by setting
> 
> OPTS+=" --model-parallel"
> OPTS+=" --model-parallel-size ${MP_SIZE}"

Thank you! I have solved the problem. I would also like to ask: when using Qwen models for supervised fine-tuning (SFT) on my custom dataset, the validation loss first decreases and then increases, and both the teacher model and student model exhibit repetitive generation patterns. How did you resolve this issue?

**Comment by t1101675** [comment]
You can increase the [temperature](https://github.com/microsoft/LMOps/blob/5b9089087d25f940e52bc8d113c39d28403445b1/minillm/arguments.py#L183) or use a [repetition penalty](https://github.com/microsoft/LMOps/blob/5b9089087d25f940e52bc8d113c39d28403445b1/minillm/arguments.py#L180) in generation
